app/controllers/users_controller.rb

```diff
@@ -34,6 +36,20 @@ def destroy
   def index
   end
   
+  # Confirms a logged-in user.
+  def logged_in_user
+    unless logged_in?
+      flash[:danger] = "Please log in."
+      redirect_to login_path
+    end
+  end
+  
+  # Confirms the correct user.
+  def correct_user
+    @user = User.find(params[:id])
```

Comment:
No I was referring to `correct_user` but I take back what I said. It makes sense that you wouldn't allow me to say
```bash
curl -X DELETE '/users/<ryan-kwon-id>
```
That said, I can foresee you reusing this filter in other methods, so consider moving it to `ApplicationController`.